apps/nginx/management/commands/https.py

```
@@ -0,0 +1,225 @@
+                cert_summary = self._verify_certificate(cert, sudo=sudo)
+            self.stdout.write(
+                f"- {config.name}: protocol={config.protocol}, enabled={config.enabled}, "
+                f"certificate={cert_label}"
+            )
+            if cert_summary:
+                self.stdout.write(f"  - {cert_summary}")
+
+    def _verify_certificate(self, cert, *, sudo: str) -> str:
+        try:
+            result = cert.verify(sudo=sudo)
+        except Exception as exc:  # pragma: no cover - depends on system
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)